leku/src/main/res/drawable/leku_select_location_button_background.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle"
+    >
+  <solid android:color="?colorPrimary"/>
+  <corners android:radius="8dp"/>
```

Comment:
I think the bottom corners should be `radius=0`. Now we can see a rounded corners behind the Navbar.
```suggestion
  <corners
      android:bottomLeftRadius="0dp"
      android:bottomRightRadius="0dp"
      android:topLeftRadius="8dp"
      android:topRightRadius="8dp"
      />
```

Reply:
You are right. Thanks!